src/main/templates/partials/header.hbs

```diff
@@ -15,6 +15,7 @@
   <script src="https://code.jquery.com/jquery-3.5.1.js"></script>
   <script src="https://cdn.datatables.net/1.10.22/js/jquery.dataTables.min.js"></script>
   <script src="https://cdn.datatables.net/fixedheader/3.1.7/js/dataTables.fixedHeader.min.js"></script>
+  <script>document.domain='isdcf.com';</script>
```

Comment:
Add comment that mentions that this is specific to a WP plugin

Reply:
Commend added to code and to this PR.